submodules/0001-VST3-cmake_Add-RelWithDebInfo-and-MinSizeRel-configs.patch

```diff
@@ -0,0 +1,142 @@
+From 24436f07be9fe055f6cd0f8ab65bf1798ff41a9c Mon Sep 17 00:00:00 2001
+From: Matt Firth <matt.firth@bbc.co.uk>
+Date: Thu, 10 Feb 2022 16:21:11 +0000
+Subject: [PATCH 1/2] Add RelWithDebInfo and MinSizeRel configs
+
+---
+ modules/SMTG_AddSMTGLibrary.cmake          | 12 ++++++------
+ modules/SMTG_AddVST3Library.cmake          |  2 +-
+ modules/SMTG_ConfigureCmakeGenerator.cmake |  2 +-
+ modules/SMTG_PlatformToolset.cmake         | 16 ++++++++--------
+ modules/SMTG_UniversalBinary.cmake         |  2 +-
+ 5 files changed, 17 insertions(+), 17 deletions(-)
+
+diff --git a/modules/SMTG_AddSMTGLibrary.cmake b/modules/SMTG_AddSMTGLibrary.cmake
+index 2cbf062..4716b02 100644
+--- a/modules/SMTG_AddSMTGLibrary.cmake
++++ b/modules/SMTG_AddSMTGLibrary.cmake
+@@ -116,7 +116,7 @@ function(smtg_create_link_to_plugin target)
+             COMMAND mklink /D
+                 ${SRC_NATIVE_PATH}
+                 "$<$<CONFIG:Debug>:${TARGET_DESTINATION_DEBUG}>"
+-                "$<$<CONFIG:Release>:${TARGET_DESTINATION_RELEASE}>"
++                "$<$<CONFIG:Release,RelWithDebInfo,MinSizeRel>:${TARGET_DESTINATION_RELEASE}>"
```

Comment:
Probably not an issue as I don't think we actually hit this code, but should we have different target destinations for the different configs here?